modules/artifact/src/test/java/org/mule/runtime/module/artifact/classloader/FilteringArtifactClassLoaderTestCase.java

```diff
@@ -146,15 +166,31 @@ public void getsExportedResources() throws Exception {
     when(filter.exportsResource(RESOURCE_NAME)).thenReturn(true);
     when(artifactClassLoader.findResources(RESOURCE_NAME)).thenReturn(new EnumerationAdapter<>(Collections.singleton(resource)));
 
-    filteringArtifactClassLoader = doCreateClassLoader();
+    filteringArtifactClassLoader = doCreateClassLoader(emptyList());
 
     Enumeration<URL> resources = filteringArtifactClassLoader.getResources(RESOURCE_NAME);
     assertThat(resources, EnumerationMatcher.equalTo(Collections.singletonList(resource)));
   }
 
   @Test
+  public void loadsExportedServices() throws ClassNotFoundException, IOException {
+    URL expectedResource = new URL("file:///app.txt");
+
+    filteringArtifactClassLoader =
+        doCreateClassLoader(singletonList(new ExportedService(SERVICE_INTERFACE_NAME, expectedResource)));
+
+    URL resource = filteringArtifactClassLoader.getResource(SERVICE_RESOURCE_NAME);
+
+    Enumeration<URL> resources = filteringArtifactClassLoader.getResources(SERVICE_RESOURCE_NAME);
+    assertThat(resources, EnumerationMatcher.equalTo(Collections.singletonList(resource)));
```

Comment:
static import. Convert the Enumeration to a List for assrtion (`Collections.list`)